app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,27 @@ def show
     @user = @post.author
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @new_post = current_user.posts.new(post_params)
+    respond_to do |f|
+      f.html do
+        if @new_post.save
+          redirect_to "/users/#{@new_post.author.id}/posts/"
+        else
+          render :new
+        end
+      end
+    end
+  end
+
+  private
+
+  def post_params
+    params.require(:post).permit(:title, :text)
```

Comment:
- [ ] I'm getting an error while creating `post` since you have used a different variable name here for `title` in comparison to the database schema. Please fix it.